ledger/ledger-api-domain/src/main/scala/com/digitalasset/ledger/api/DeduplicationPeriod.scala

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package com.daml.ledger.participant.state.v2
+package com.daml.ledger.api
```

Comment:
Where is this going to be used as part of the API. If it is only as part of command submission, I am wondering whether this shouldn't be in `com.daml.ledger.api.messages.command.completion`. l

Reply:
Yes, so in the API it's gonna be part of `com.daml.ledger.api.Commands`. i'm up for moving it anywhere as I am not really familiar with the structure we have here

Reply:
`com.daml.ledger.api.messages.*` seems to only contain types corresponding to top-level messages used in the ledger API.